alf/algorithms/rl_algorithm.py

```diff
@@ -846,11 +894,12 @@ def _train_iter_off_policy(self):
             return 0
 
         self.train()
-        steps = self.train_from_replay_buffer(update_global_counter=True)
-
-        if unrolled:
-            with record_time("time/after_train_iter"):
-                self.after_train_iter(root_inputs, rollout_info)
+        steps = 0
+        for i in range(effective_unroll_iters):
```

Comment:
it's possible the effective_unroll_iters is always smaller than 1 in the case of num_envs > 1.

Reply:
Now also handles the fractional unroll case.